packages/block-editor/src/components/contrast-checker/index.js

```diff
@@ -2,10 +2,12 @@
  * External dependencies
  */
 import tinycolor from 'tinycolor2';
+import React, { useEffect } from 'react';
```

Comment:
We use @wordpress/element as an abstraction on top of `React`, can you replace this line with `import { useEffect } from @wordpress/element;` and move it to the WP dependencies section?